defrag/modules/news.py

```diff
@@ -0,0 +1,91 @@
+# Defrag - centralized API for the openSUSE Infrastructure
+# Copyright (C) 2021 openSUSE contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Dict, List, NamedTuple, Optional
+
+from aiohttp.client import ClientSession
+from defusedxml.ElementTree import fromstring
+from collections import namedtuple
+import asyncio
+import aiohttp
+
+__MOD_NAME__ = "news"
```

Comment:
I think I would call this one "Reddit", since we also have news.opensuse.org

Reply:
* [x]